package.json

```diff
@@ -29,8 +29,27 @@
     "handler",
     "exit"
   ],
+  "xo": {
+    "esnext": true
+  },
+  "config": {
+    "nyc": {
+      "exclude": [
+        "node_modules/",
+        "test.js",
+        "fixture.js"
+      ]
+    }
+  },
   "devDependencies": {
-    "ava": "*",
-    "xo": "*"
+    "ava": "sindresorhus/ava",
+    "babel-runtime": "^5.8.29",
```

Comment:
Need the latest ava for coverage to work.

Need babel-runtime because latest ava is broken without it.

We can bump both when ava `0.4.0` is out.